src/components/Body/Body.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { useUser } from '../../context/UserContext';
+import { useEntryList } from '../../context/EntryContext';
+import useLocalStorage from '../../hooks/useLocalStorage/useLocalStorage';
+
+const Body = () => {
+  const { entryList, setEntryList } = useEntryList();
+  const { user, setUser } = useUser();
+  const [userName, setUserName] = useLocalStorage('name', '');
+  const [userEntry, setUserEntry] = useState('');
+
+  function updateEntryList() {
+    if (!userEntry) return;
+    setUser(userName);
+    setEntryList([...entryList, { userName, userEntry: userEntry }]);
```

Comment:
you might want to use a callback and spread in prevState instead of entryList here to ensure your state is properly updated